src/commands/synthetics/run-tests-lib.ts

```diff
@@ -252,11 +252,6 @@ export const executeWithDetails = async (
     ...runConfig,
   }
 
-  // We don't want to have default globs in case suites are given.
-  if (!runConfig.files && suites?.length) {
-    localConfig.files = []
-  }
```

Comment:
We remove this because we don't have a default glob anymore right? 🤔

Reply:
Yep, indeed 👍